tools/wpt/revlist.py

```diff
@@ -0,0 +1,167 @@
+import argparse
+import calendar
+import logging
+import os
+import time
+from tools.wpt.testfiles import get_git_cmd
+
+here = os.path.dirname(__file__)
+wpt_root = os.path.abspath(os.path.join(here, os.pardir, os.pardir))
+
+logger = logging.getLogger()
+
+DEFAULT_EPOCH = 86400
+DEFAULT_EPOCH_THESHOLD = 600
+DEFAULT_EPOCH_UNTIL = calendar.timegm(time.gmtime())
+DEFAULT_MAX_COUNT = 1
+
+FOR_EACH_REF_MAX_COUNT = 100000
+
+TAGGED_REVISIONS = "refs/tags/merge_pr_*"
+
+WEEKLY_EPOCH_SIZE = 604800
+WEEKLY_EPOCH_OFFSET = -259200  # Monday, 29 December 1969 0:00:00
+
+
+MYPY = False
+if MYPY:
+    # MYPY is set to True when run under Mypy.
+    from typing import Any
+    from typing import Dict
+    from typing import List
+    from typing import Text
+
+
+def parser_epoch(**kwargs):
+    # type: (**Any) -> int
+    epoch = kwargs.get("epoch", DEFAULT_EPOCH)
+    if isinstance(epoch, int):
+        return epoch
+    if epoch.lower() == "three-hourly":
+        return 10800
+    if epoch.lower() == "six-hourly":
+        return 21600
+    if epoch.lower() == "twelve-hourly":
+        return 43200
+    if epoch.lower() == "daily":
+        return 86400
+    if epoch.lower() == "weekly":
+        return 604800
+    return int(epoch)
+
+
+def parser_max_count(**kwargs):
+    # type: (**Any) -> int
+    max_count = kwargs.get("max_count", DEFAULT_MAX_COUNT)
+    return int(max_count)
+
+
+COMMAND_ARGS = {
```

Comment:
Can you inline this whole block into `get_parser`, like for this command?

https://github.com/web-platform-tests/wpt/blob/48d921831f4b790ae46bbd9b102bdb7235f0a201/tools/wpt/testfiles.py#L342-L364

Because `run_rev_list` is called with the parsed `kwargs` you shouldn't need to reference `COMMAND_ARGS` (or `get_parser`) anywhere else. Instead, just reference `kwargs["epoch"]` and `kwargs["max_count"]`.